systemtest/010-inspect.bats

```diff
@@ -32,7 +32,8 @@ load helpers
     config_digest=$(jq -r '.config.digest' <<<"$inspect_local_raw")
 
     # Each SHA-named layer file (but not the config) must be listed in the output of 'inspect'.
-    # As of Skopeo 1.6, (skopeo inspect)'s output lists layer digests,
+    # In all existing versions of Skopeo (with 1.6 being the current as of this comment),
```

Comment:
This is hopefully comprehensible, but it feels clumsy to me. Suggestions to reword are welcome.